piws/site_cubicweb.py

```diff
@@ -44,4 +44,11 @@
       'group': 'piws',
       'level': 1,
       }),
+    ("enable-cwusers-watcher", {
+        "type": "string",
+        "default": None,
+        "help": "Enable CWUsers watcher with email notification.",
```

Comment:
More information:
If this option is selected, an email is sent (this email address  has to be set in the [MAIL] all-in-one section) when a CW user is created or deleted.